### DISCUSSION TITLE
i'm facing this issue, how do i fix it?

### DISCUSSION THREAD

**Discussion by zackomar8** [sharafdin/yonode#357] [Q&A] [answered]
![WhatsApp Image 2024-04-26 at 17 38 29](https://github.com/sharafdin/yonode/assets/146327762/44adca0b-8ca2-4001-b67b-16de7792a06d)

**Comment by hanad124** [comment, marked as answer]
To address the issue, you need to make sure that the `yonode` command is cloning your project into a new directory that doesn't already exist, or if it does, it must be empty. If you want to use the same 'yonode-app' directory, you must first ensure that it is empty. Here's what you can do:

- If you want to clear out the existing 'yonode-app' directory, you can do so by navigating to the directory in your terminal and using the command `rm -rf *` (on Unix-based systems) or `del /F /Q *` (on Windows in a CMD prompt) to remove all files within it. Be very cautious with these commands, as they will delete all the files in the directory without asking for confirmation.

- If you want to use a different directory name that doesn't already exist, you can simply specify a new name when you run the `yonode` command. For example, if you want to name the directory 'new-yonode-app', you would run:

  ```shell
  npx yonode@latest yonode-app new-yonode-app

**Reply by zackomar8** [reply]
thanks, i will try it.

**Reply by zackomar8** [reply]
thanks it worked for me 🙏
